config/kafka-sizes-configuration.yaml

```diff
@@ -0,0 +1,41 @@
+# This configuration defines a list of supported Kafka instance profile and their sizes for KAS Fleet Manager. 
+# Modify this file to add additional Kafka instance profiles or sizes or change the values of the default profiles listed below.
+# 
+# The following properties must be defined for each Kafka profile:
+#   - id: Identifier for the Kafka profile. Each profile name should be unique.
+#   - sizes: A list of Kafka instance sizes available for this profile (should not be an empty list)
+#
+# The following properties must be defined for each Kafka instance size (all values must be larger than '0'):
+#   - id: The Kafka instance size identifier. Each size id should be unique.
+#   - ingressThroughputPerSec: Maximum limit for ingress throughput per second (MB/s).
+#   - egressThroughputPerSec: Maximum limit for egress throughput per second (MB/s).
+#   - totalMaxConnections: Maximum client connections allowed.
+#   - maxConnectionAttemptsPerSec: Maximum client connection attempts per second.
+#   - maxDataRetentionSize: Maximum storage (GB).
+#   - maxDataRetentionPeriod: Maximum duration(ISO8601) for retaining data.
+#   - maxPartitions: Maximum partitions for all topics combined.
+#   - cost: Quota cost for selecting this size for a Kafka instance.
+---
+supported_kafka_sizes:
```

Comment:
I believe this should be `supported_instance_types`

Reply:
this file should (and will shortly) be removed